# The detected class
int8 class_id

# The region of interest within the image
sensor_msgs/RegionOfInterest roi

# The detection confidence
float32 confidence
